igcommit/config.py

```diff
@@ -133,10 +133,11 @@
 
 # Ruby
 checks.append(CheckCommand(
-    args=['rubocop', '--format=emacs', '--stdin', '/dev/stdin'],
+    args=['rubocop', '--format=emacs', '--stdin'],
     extension='rb',
     exe_pattern=file_extensions['rb'],
     config_files=[CommittedFile('.rubocop.yml')],
+    append_filepath=True,
```

Comment:
I don't see how this would work.  Pre-receive hook operates on a bare Git repository.  It won't have the checked out copy of the source files.

Reply:
it doesn't need a checked out file, rubocop still receives the contents via stdin (`--stdin`). `/dev/stdin` as argument was simply wrong. it won't actually load that file, it just uses the name to check against `Exclude` directives

Reply:
Got it.  Cool.  Maybe comment this somewhere in the code.

Reply:
comment looking good?

Reply:
Great